include/cpr/auth.h

```diff
@@ -9,18 +9,14 @@ namespace cpr {
 
 class Authentication {
   public:
-    Authentication(std::string&& username, std::string&& password)
```

Comment:
Please also add rvalue constructors for `Authentication` to keep compatibility with older implementations.

Reply:
Rvalues can be passed to const lvalue reference arguments, so this does not break compatibility.

Reply:
You are right.
Nevertheless, I'd like to keep the `rvalue` constructors for verbosity reasons.

Reply:
Done.